kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/GwtComponentServiceImpl.java

```diff
@@ -59,6 +62,8 @@
     private static final String SERVICE_FACTORY_PID = "service.factoryPid";
     private static final String KURA_UI_SERVICE_HIDE = "kura.ui.service.hide";
 
+    private static final int SERVICE_WAIT_TIMEOUT = 60000;
```

Comment:
I believe the appropriate solution is to have the constant variable in the inline method itself. Other methods won't be able to see it at all as it is only used in one place. If we consider encapsulation and clean code approach, if we define the constant in class level, you get a long list of class variables as well. In addition, we have to consider the unnecessary memory footprints as well. If the constant is inline, the variables are destroyed but in case of static constant, it will remain in memory all the time.